ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -19,19 +19,19 @@ Control {
     id: root
 
     property alias currentTabIndex: stackLayout.currentIndex
-
-    property string mainDisplayName
-    property bool readOnly
-    property var profileStore
-    property var walletStore
-    property var networkConnectionStore
-
-    property bool livePreview: false
-    property var livePreviewValues: ({})
-
+    
+    property alias communitiesModel: communitiesView.model
+    property alias accountsModel: accountsView.model
+    property alias collectiblesModel: collectiblesView.model
+    property alias assetsModel: assetsView.model
+    //property alias socialLinksModel: socialLinksView.model
```

Comment:
to be added?

Reply:
yes, with the web tab. There's no alias to bind to yet.
But the model itself is available